client/coral-framework/components/Slot.js

```diff
@@ -4,9 +4,9 @@ import styles from './Slot.css';
 import {connect} from 'react-redux';
 import {getSlotElements} from 'coral-framework/helpers/plugins';
 
-function Slot ({fill, inline = false, plugin_config: config, ...rest}) {
+function Slot ({fill, inline = false, right = false, plugin_config: config, ...rest}) {
```

Comment:
Not sure if we should include all these styling into the Slot itself. If we continue this pattern someday we would have `inline, right, left, top, down, center, fixed, absolute`. Maybe just allow the `style` and `className` property to add the styling to this component from a parent instead of doing it here?